pallets/vault-registry/src/benchmarking.rs

```diff
@@ -151,6 +151,14 @@ benchmarks! {
 		Oracle::<T>::_set_exchange_rate(vault_id.clone().account_id, get_wrapped_currency_id(), UnsignedFixedPoint::<T>::checked_from_rational(10, 1).unwrap()).unwrap();
 		VaultRegistry::<T>::liquidate_vault(&vault_id).unwrap();
 	}: _(RawOrigin::Signed(vault_id.account_id), vault_id.currencies.clone())
+
+	set_punishment_delay {
+		let punishment_delay : T::BlockNumber = T::BlockNumber::default();
```

Comment:
Nit: There should not be a space character after `let punishment_delay`

Reply:
Done